config/main

```diff
@@ -0,0 +1 @@
+include config/default
```

Comment:
Why not having one makefile per developper ?
This is consitent with other projects and like this it is easy to override only some settings and it is easy to see what is overridden.